portal/views/procedure.py

```diff
@@ -122,8 +122,12 @@ def post_procedure():
     # patient_id must first be parsed from the JSON subject field
     # standard role check is below after parse
 
-    if not request.json or 'resourceType' not in request.json or\
-            request.json['resourceType'] != 'Procedure':
+    if (
+        any([
```

Comment:
Can you use a tuple (instead of a list) and consolidate the `if` and `any(` line?

Reply:
I believe consolidating the ``if`` and ``any(`` lines causes double-indenting to be required (or violating pep8)